doc/src/manual/performance-tips.md

```diff
@@ -599,8 +546,7 @@ optimize the body of the loop. There are several possible fixes:
 
   * Initialize `x` with `x = 1.0`
   * Declare the type of `x`: `x::Float64 = 1`
-  * Use an explicit conversion: `x = oneunit(T)`
-  * Initialize with the first loop iteration, to `x = 1/bar()`, then loop `for i = 2:10`
+  * Initialize with the first loop iteration, to `x = 1 / rand()`, then loop `for i = 2:10`
```

Comment:
`rand` seems even more confusing than `bar` here. Though this section clearly needs to be developed more.

Reply:
With `bar` it wasn't even the case that the result had to be a float so that's why I changed it.

Reply:
OK. That section isn't completely well constructed anyway...